### PR TITLE
- Add VMM as a memory model

### PR DESCRIPTION
**Pull request by hernanponcedeleon**
- Update dartagnan options
- Remove mocking code for dartagnan checker

### CODE REVIEW

File: checker/checker_dartagnan.go [hunk 1]
```diff
@@ -81,8 +76,7 @@ func (c *DartagnanChecker) run(ctx context.Context, testFn string) (string, erro
 	opts := []string{
 		"--property=program_spec,liveness",
 		"--modeling.threadCreateAlwaysSucceeds=true",
-		"--modeling.mixedType=true",
```
[the remark this answers is not in the document]
Comment (hernanponcedeleon) [reply checker/checker_dartagnan.go:84]: The option was removed in version 4.0.0

File: checker/checker_dartagnan.go [hunk 2]
```diff
@@ -193,8 +187,4 @@ func (c *DartagnanChecker) createVerifierMock() (string, error) {
 const dartagnanVerifierMock = `
 #include <pthread.h>
 
-int __VERIFIER_thread_create(pthread_t *thread, const pthread_attr_t *attr,
```
[the remark this answers is not in the document]
Comment (hernanponcedeleon) [reply checker/checker_dartagnan.go:196]: Actually, method `createVerifierMock()` can also be removed. Will do

**Comment by db7** [comment]
Great. Thanks.

**Review by db7 — APPROVED** [review]

**Comment by db7** [comment]
@hernanponcedeleon you have to sign the commits.

**Pull request merged**